docs/kcl/std.json

```diff
@@ -153550,25 +153550,79 @@
     "args": [
       {
         "name": "num",
-        "type": "i64",
+        "type": "NumberArg",
         "schema": {
           "$schema": "https://spec.openapis.org/oas/3.0/schema/2019-04-02#/definitions/Schema",
-          "title": "int64",
-          "type": "integer",
-          "format": "int64"
+          "title": "NumberArg",
```

Comment:
Can we call it `Number` ?

Reply:
No because it collides with the `Number` variant of `KclValue`, and the docs generator isn't smart enough to distinguish between the two.

Reply:
No longer relevant since switching to f64.